File: src/main/java/com/example/installcertatruntime/car/CarService.java
package com.example.installcertatruntime.car;

import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class CarService {

    private final CarRepository carRepository;

    public CarService(CarRepository carRepository) {
        this.carRepository = carRepository;
    }

    public List<Car> all() {
        return carRepository.findAll();
    }

    public Car create(Car car) {
        return carRepository.save(car);
    }
}

File: src/main/kotlin/com/example/installcertatruntime/car/CarService.kt
package com.example.installcertatruntime.car

import org.springframework.stereotype.Service

@Service
class CarService(private val carRepository: CarRepository) {

    fun all(): List<Car> {
        return carRepository.findAll()
    }

    fun create(car: Car): Car {
        return carRepository.save(car)
    }
}
